rest_framework_filters/backends.py

```diff
@@ -32,3 +33,30 @@ def filter_queryset(self, request, queryset, view):
         # us to avoid maintenance issues with code duplication.
         with self.patched_filter_class(request):
             return super(DjangoFilterBackend, self).filter_queryset(request, queryset, view)
+
+    def get_schema_fields(self, view):
```

Comment:
This is the incorrect method to implement, as this is based on the deprecated coreapi/coreschema libraries. You would want to implement `get_schema_operation_parameters` instead. See https://github.com/carltongibson/django-filter/pull/1086

Reply:
My code is for branch 0.11x which currently works with django-filter 1.1 so I am not sure the new method was implemented there. I will check it and update as needed.